src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -604,44 +604,120 @@ internal BoundStatement BindLocalDeclarationStatement(LocalDeclarationStatementS
 
         private BoundStatement BindDeclarationStatementParts(LocalDeclarationStatementSyntax node, DiagnosticBag diagnostics)
         {
-            var typeSyntax = node.Declaration.Type.SkipRef(out _);
-            bool isConst = node.IsConst;
+            var variableList = node.Declaration.Variables;
+            int variableCount = variableList.Count;
 
+            bool isConst = node.IsConst;
+            var typeSyntax = node.Declaration.Type.SkipRef(out _);
             bool isVar;
             AliasSymbol alias;
             TypeSymbol declType = BindVariableType(node.Declaration, diagnostics, typeSyntax, ref isConst, isVar: out isVar, alias: out alias);
 
-            // UNDONE: "possible expression" feature for IDE
+            Conversion iDisposableConversion = default;
+            MethodSymbol disposeMethod = default;
```

Comment:
These declarations can be moved inside the `if (node.UsingKeyword != default)` block, and the initializers are not needed.